V0_Display_RP2040/STLs/README.md

```diff
@@ -0,0 +1,23 @@
+# V0 Display Enclosure #
+![V-Naught_Display_Enclosure](../Images/V0pt1_Enclosure.png )
```

Comment:
Looks like this image needs to be updated, as this doesn't exist in the current release. Looks like you just copy/pasted my original V0 display enclosure instructions without updating/copying over the needed image.